services/federator/federator.integration.yaml

```diff
@@ -7,6 +7,9 @@ federatorExternal:
 brig:
   host: 0.0.0.0
   port: 8082
+cargohold:
+  host: 0.0.0.0
+  port: 8084
```

Comment:
This block also needs to be added to the similar place under `charts/federator/` for CI to pass.